proxy/coco/src/peer/gossip.rs

```diff
@@ -8,12 +8,12 @@ use librad::{
 use crate::{oid::Oid, State};
 
 /// Announce a new rev for the `urn`.
-pub async fn announce(state: &State, urn: &RadUrn, rev: &Oid) {
+pub async fn announce(state: &State, urn: &RadUrn, rev: Option<Oid>) {
     let protocol = state.api.protocol();
     protocol
         .announce(Gossip {
             urn: urn.clone(),
-            rev: Some(Rev::Git((*rev).into())),
+            rev: rev.map(|rev| Rev::Git((rev).into())),
```

Comment:
What's the reason for doing this? :thinking:

Reply:
Doing what?

Reply:
Right sorry, I mean what's the reason for changing this to always announce the `Rev` to optionally announcing it?

Reply:
We changed the function signature to take an `Option`, so we won't always hav a rev to announce. This was necessary to support the proect creation case where we solely interested in the urn and nothing else.

Reply:
Ah makes sense, perfect :ok_hand: